src/sagemaker/clarify.py

```diff
@@ -2236,23 +2481,45 @@ def _add_methods(
     def _merge_explainability_configs(
         cls,
         explainability_config: Union[ExplainabilityConfig, List[ExplainabilityConfig]],
+        time_series_case: bool = False,
     ):
         """Merges explainability configs, when more than one."""
         if isinstance(explainability_config, list):
             explainability_methods = {}
             if len(explainability_config) == 0:
                 raise ValueError("Please provide at least one explainability config.")
             for config in explainability_config:
+                # ensure all provided explainability configs
+                # are AsymmetricSHAPConfig in time series case
+                is_asym_shap_config = isinstance(config, AsymmetricSHAPConfig)
+                if time_series_case and not is_asym_shap_config:
```

Comment:
Suggest to run all validations at the entry point.

Reply:
Will address today.